tests/misc/BuildManPageTest.py

```diff
@@ -75,8 +75,8 @@ def test_mk_options(self):
     def test_mk_footer(self):
         uut = ManPageFormatter(app_name, ext_sections=sections)
         self.assertEqual(uut._mk_footer(),
-                         ".SH {0}\n"
-                         " {1}".format(section_name.upper(), section_text))
+                         ".SH {}\n {}".format(
```

Comment:
@Makman2  Naah it's fine. It looks actually better now.  :+1:
I just wanted to confirm. Because, I happen to be in such situation often when I need to choose between `convention` and `asthetics` :D